application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/LicensesNotificationsUIX.xml

```diff
@@ -183,14 +326,26 @@
     <property>
       <code>#template('colorThemeInit.vm')
 
-#tmNotifications &gt; .dropdown-toggle span:empty::before {
+.error-notification span:empty::before {
```

Comment:
`error-notification` is too generic. Same for warning

Reply:
`licenses-error` / `licenses-warning` looks ok, since this is only for the notification bell. And the other class contain the `notification` word since it holds the actual notification